docs/contents/components/forms/editable/accessibility.ja.mdx

```diff
@@ -0,0 +1,21 @@
+---
+title: Editable
+description: "`Editable`は、インラインで編集可能なテキスト入力を取得するために使用されるコンポーネントです。"
+package_name: "@yamada-ui/editable"
+with_description: true
+order: 3
+tab: アクセシビリティ
+---
+
+## ARIAロールと属性
+
+| コンポーネント     | ロールと属性    | 使い方                                             |
+| ------------------ | --------------- | -------------------------------------------------- |
+| `EditableInput`    | `aria-invalid`  | Editableがinvalidの場合に`"true"`が設定されます。  |
```

Comment:
Change "Editable" to "`Editable`".

Change

```
Editableがinvalidの場合に`"true"`が設定されます。
```

to

```
`Editable`に`isInvalid`が設定されている場合は`"true"`を設定します。
```

Other sentences will be modified in the same way.

Reply:
fixed

Reply:
fixed